### PR TITLE
Add 18.04 server tutorial

### PR DESCRIPTION
**Pull request by evilnick**
also moves existing tutorial and renames it to reference 16.04

### CODE REVIEW

**Comment by evilnick** [comment]
@degville apologies for the nasty diff, that's what happens when you move things!

File: tutorials/server/install-ubuntu-server/install-ubuntu-server.md [hunk 1]
```diff
@@ -16,14 +16,14 @@ author: Canonical Web Team <webteam@canonical.com>
 ## Overview
 Duration: 0:01
 
-Ubuntu Server is Ubuntu tailored for networks and services, and it's just as capable sharing files on your home network as it is running within a Hadoop cluster.
+Ubuntu Server is a variant of the standard Ubuntu you already know, tailored for networks and services. It's just as capable of running a simple file server or operating within a 50,000 node cloud.
```
Comment (degville) [inline tutorials/server/install-ubuntu-server/install-ubuntu-server.md]: The *or* doesn't sound quite right with the *just as capable*

File: tutorials/server/install-ubuntu-server/install-ubuntu-server.md [hunk 2]
```diff
@@ -35,126 +35,182 @@ You'll need to consider the following before starting the installation:
...
+Most computers will automatically boot from USB or DVD, though in some cases this is disabled to improve boot times. If you don't see the boot message and the "Welcome" screen which should appear after it, you will need to set your computer to boot from the install media.
 
-## Boot from USB flash drive
-Duration: 0:02
+There should be an on-screen message when the computer starts telling you what key to press for settings or a boot menu. Depending on the manufacturer, this could be `Escape`, `F2`,`F10` or `F12`. Simply restart your computer and hold down this key until the boot menu appears, then select the drive with the Ubuntu install media.
 
-Most computers will boot from USB automatically. Simply insert the USB flash drive and either power on your computer or restart it. You should see the same language menu we saw in the previous 'Install from DVD' step, followed by the boot menu.
+If you are still having problems, check out the [Ubuntu Community documentation on booting from CD/DVD][docs-dvd] page.
```
Comment (degville) [inline tutorials/server/install-ubuntu-server/install-ubuntu-server.md]: not sure *page* is necessary.

File: tutorials/server/install-ubuntu-server/install-ubuntu-server.md [hunk 3]
```diff
@@ -35,126 +35,182 @@ You'll need to consider the following before starting the installation:
...
 
-If your computer doesn't automatically boot from USB, try holding `F12` when your computer first starts. With most machines, this will allow you to select the USB device from a system-specific boot menu.
+## Chose your language
```
Comment (degville) [inline tutorials/server/install-ubuntu-server/install-ubuntu-server.md]: choose

File: tutorials/server/install-ubuntu-server/install-ubuntu-server.md [hunk 4]
```diff
@@ -35,126 +35,182 @@ You'll need to consider the following before starting the installation:
...
 
-![screenshot](https://assets.ubuntu.com/v1/cfd45d64-server-password.png)
+Now we are ready to select what you want to install. There are three options in the menu:
 
-## Storage configuration
-Duration: 0:05
+![screenshot][menu-install]
 
-After answering a question about your time zone, you need to configure local storage.
+The bottom two options are used for installing specific components of a Metal As A Service (MAAS) install. If you are installing MAAS, you should check out the [MAAS documentation][docs-maas] for more information on this! The rest of this tutorial assumes you select the first option, `Install Ubuntu`
```
Comment (degville) [inline tutorials/server/install-ubuntu-server/install-ubuntu-server.md]: missing a full stop at the end.

**Pull request merged**